ui-tests/src/test/java/io/syndesis/qe/steps/connections/ConnectionSteps.java

```diff
@@ -17,108 +16,63 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
-import java.util.Optional;
 
 import cucumber.api.java.en.Then;
 import cucumber.api.java.en.When;
-import io.syndesis.qe.accounts.Account;
-import io.syndesis.qe.accounts.AccountsDirectory;
-import io.syndesis.qe.pages.connections.detail.ConnectionDetailPage;
-import io.syndesis.qe.pages.connections.edit.ConnectionCreatePage;
-import io.syndesis.qe.pages.connections.list.ConnectionListPage;
-import io.syndesis.qe.pages.connections.list.ConnectionsListComponent;
+import io.syndesis.qe.fragments.common.menu.KebabMenu;
+import io.syndesis.qe.pages.connections.Connections;
 import lombok.extern.slf4j.Slf4j;
 
-/**
- * Created by sveres on 11/10/17.
- */
 @Slf4j
 public class ConnectionSteps {
 
-    private ConnectionCreatePage editPage = new ConnectionCreatePage();
-    private ConnectionDetailPage detailPage = new ConnectionDetailPage();
-    private ConnectionListPage listPage = new ConnectionListPage();
+    private Connections connectionsPage = new Connections();
 
-    @Then("^Camilla is presented with \"([^\"]*)\" connection details")
-    public void verifyConnectionDetails(String connectionName) {
-        log.info("Connection detail page must show connection name");
-        assertThat(detailPage.connectionName(), is(connectionName));
+    @When("^opens the \"([^\"]*)\" connection detail$")
```

Comment:
Maybe ` @When("^.*opens? the \"([^\"]*)\" connection detail$")`

Reply:
changed